src/ui/window.cpp

```diff
@@ -55,6 +55,29 @@ class WindowTitleLabel : public Label {
     setType(kWindowTitleLabelWidget);
     initTheme();
   }
+private:
+  void onSetDecorativeWidgetBounds() override {
+    if (theme())
+      theme()->setDecorativeWidgetBounds(this);
```

Comment:
We can replace these two lines by a call to Label::onSetDecorativeWidgetBounds(); which basically calls Widget::onSetDecorativeWidgetBounds() and does the same as these two lines.